build.gradle.kts

```diff
@@ -105,7 +107,9 @@ dependencies {
 
   implementation("com.fasterxml.jackson.module:jackson-module-kotlin:2.10.2")
 
-  testImplementation("org.springframework.boot:spring-boot-starter-test")
+  testImplementation("org.springframework.boot:spring-boot-starter-test") {
+    exclude("org.junit.vintage", "junit-vintage-engine")
```

Comment:
bit annoying that default is to include junit 4 so have to explicitly exclude

Reply:
why do you need to exclude? Don't you just not use junit4?

Reply:
Agreed, but far too easy to do `@Test` and not realise that it has automatically included the junit 4 version instead by accident.  This way that won't happen.